content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -99,6 +99,258 @@ Printer`. Follow the steps and select the *make* and *model* corresponding to th
+
+  - `SCALE<X>`: Scale of the picture (with the same aspect ratio). `X` should be an integer
+    describing the scale percentage that should be used.
+
+    .. example::
+       `100` is the original size, `50` is half the size, `200` is twice the size.
+
+  - `LDV`: *Low Density Vertical* (will be set to *High Density Vertical* if not specified).
+  - `LDH`: *Low Density Horizontal* (will be set to *High Density Horizontal* if not specified).
+
+    .. note::
+       *Density* parameters might need to be configured in a particular way depending on the printer
+       model.
+
+.. seealso::
+   Visit `Epson's ESC * documentation
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=88>`_ and click on the
+   printer model printer in the table above to see if the printer should set these parameters.
+
+.. example::
+   The following are examples of proper and improper name formatting:
+
+   Proper name formatting:
+
+   - `EPSONTMm30II__IMC__`
+   - `EPSON_TM_U220__IMC_LDV_LDH_SCALE80__`
+
+   Improper name formatting (this will not prevent printing, but the result might not have the
+   expected printed output):
+
+   - `EPSON TMm 30II` -> The name can't have spaces.
+   - `EPSONTMm30II` -> The name itself is correct, but it won't use `ESC *`.
+   - `EPSONTMm30II__IMC` -> This name is missing the end `__`.
+   - `EPSONTMm30II__IMC_XDV__` -> The parameter `XDV` does not match any existing parameters.
+   - `EPSONTMm30II__IMC_SCALE__` -> The parameter `SCALE` is missing the scale value.
+
+Finish adding a printer
+"""""""""""""""""""""""
+
+After setting the name of the printer with the appropriate naming convention, click
+:guilabel:`Continue`. Next, set the :guilabel:`Make` value to :guilabel:`Raw` and for the
+:guilabel:`Model` value, set this to :guilabel:`Raw Queue (en)`.
+
+After completing these steps, click :guilabel:`Add Printer`. If everything was done correctly, the
+page should redirect to the *Banners* page.
+
+At this point the printer should have been created, now the :abbr:`IoT (Internet of Things)` box
+just needs to detect it and then sync to Odoo's server (this could take a few minutes).
+
+Adding the printer to Odoo PoS
+""""""""""""""""""""""""""""""
+
+Once the printer is visible on the Odoo database, don't forget to choose it in the :abbr:`PoS (Point
+of Sale)`configuration as the :abbr:`IoT (Internet of Things)` printer. Navigate to
+:menuselection:`Pos App --> Settings --> Connected Devices --> IoT Box --> Receipt Printer -->
+Save`.
+
+.. note::
+   If the printer was set up incorrectly (it is still printing random text or the printed receipt is
+   too big or small) it cannot be modified via the printer name with *CUPS*. Instead, the above
+   process can be repeated to set up another printer from scratch to create one with modified
+   parameters.
+
+Example setup of the Epson TM-U220B printer using ESC *
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""
```

Comment:
The content in this section is relevant however it's a bit ugly and breaks on mobile.

Reply:
Would like to revisit the use of quotation marks as "h7" formatting—

Typically headings only go down to h6 which are normally associated with subtext (think footer links), so we'd need to create a specific style for this if we were to use it.

As it is, out of the box, the use of quotation mark symbols is reserved to indicate [paragraph text in Sphinx](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections).

...so it might actually just be better to use **bold text** because, as it is now, the `"` formatting doesn't render anything  visually unique enough to say to the reader "hey I'm a heading!"

Reply:
Also we cannot merge the doc with invalid characters like this as the error will propagate to all other versions of the doc across multiple versions (in effect, replicating an error).

Will replace with **bold text** for now.